docker/envs/.env.docker.devnet

```
@@ -1,6 +1,22 @@
+# ENSIndexer and ENSApi
+ENSINDEXER_SCHEMA_NAME=docker_devnet_v1
+# ENSIndexer and ENSApi
+RPC_URL_1=http://devnet:8545
+# ENSIndexer and ENSRainbow
 LABEL_SET_VERSION=0
+# ENSIndexer and ENSRainbow
+LABEL_SET_ID=ens-test-env
+# ENSRainbow
 DB_SCHEMA_VERSION=3
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)